app/tests/Core/documentTests.js

```diff
@@ -1367,17 +1368,21 @@ export default [
 
   // Fails "6.1.26 Prohibited Document Category Name"
   ...[
+    'Csaf_a',
     'Security_Incident_Response',
     'Informational Advisory',
     'security-incident-response',
     'Security      Advisory',
     'veX',
+    'V_ex',
+    'V___eX',
     'Informational - Advisory',
     'security-_ incident-response',
     'Security\tAdvisory',
     'Security\nAdvisory',
     'Security\rAdvisory',
   ].map((category) => ({
+    //    title: `Fails "6.1.26 Prohibited Document Category Name" (category "${category}")`,
```

Comment:
@domachine: Is it easily possible to add the name for the test (like this)? Or does that require a major restructure?

Reply:
I think this should be possible right now